### PR TITLE
rbenv: add Indonesian translation

### PR DESCRIPTION
**Pull request by apraditya**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by theodorusclarence — APPROVED** [review]

File: pages.id/common/rbenv.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# rbenv
+
+> Alat untuk menginstal versi Ruby dan mengatur lingkungan aplikasi dengan mudah.
+> Informasi lebih lanjut: <https://github.com/rbenv/rbenv>.
+
+- Instal suatu versi Ruby:
+
+`rbenv install {{versi}}`
+
+- Menampilkan daftar versi-versi stabil terbaru dari Ruby:
```
Comment (blueskyson) [inline pages.id/common/rbenv.md:10]: Is `versi-versi` a typo?

Comment (navarroaxel) [reply pages.id/common/rbenv.md:10]: No, it's a plural.

File: pages.id/common/rbenv.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`rbenv install --list`
+
+- Menampilkan daftar versi-versi Ruby yang terinstal:
```
Comment (navarroaxel) [inline pages.id/common/rbenv.md:14]: I don't speak the language, but I'm not sure about the verbs. The verbs should be in imperative mood.

File: pages.id/common/rbenv.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`rbenv install --list`
+
+- Tampilkan daftar versi-versi Ruby yang terinstal:
+
+`rbenv versions`
+
+- Gunakan spesifik versi Ruby di seluruh sistem:
+
+`rbenv global {{version}}`
```
Comment (navarroaxel) [inline pages.id/common/rbenv.md:20]:
```suggestion
`rbenv global {{versi}}`
```

File: pages.id/common/rbenv.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Gunakan spesifik versi Ruby untuk suatu direktori aplikasi/proyek:
+
+`rbenv local {{version}}`
```
Comment (navarroaxel) [inline pages.id/common/rbenv.md:24]:
```suggestion
`rbenv local {{versi}}`
```

File: pages.id/common/rbenv.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Tampilkan versi Ruby yang saat ini dipilih:
+
+`rbenv version`
+
+- Hapus sebuah versi Ruby:
+
+`rbenv uninstall {{version}}`
```
Comment (navarroaxel) [inline pages.id/common/rbenv.md:32]:
```suggestion
`rbenv uninstall {{versi}}`
```

File: pages.id/common/rbenv.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+- Tampilkan semua versi Ruby yang mengandung file yang dapat dieksekusi sesuai dengan yang disebut:
+
+`rbenv whence {{executable}}`
```
Comment (navarroaxel) [inline pages.id/common/rbenv.md:36]: could `executable` be translated?

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.id/common/rbenv.md (hunk 6 above)
Comment (apraditya) [reply pages.id/common/rbenv.md:36]: We don't have a word for `executable`. The translation is `file yang dapat dieksekusi` which doesn't fit in this context.

**Review by navarroaxel — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**